app/views.py

```diff
@@ -273,3 +273,109 @@ def admin():
         return "You cannot access this page.", 401
 
     return render_template('admin.html')
+
+
+@app.route('/projects/create', methods=['GET', 'POST'])
+def create_project():
+    if not is_logged_in():
+        if request.method == 'GET':
+            return redirect(url_for('login', next=request.url_rule))
+        elif request.method == 'POST':
+            return "Not logged in.", 401
+
+    form = forms.CreateProjectForm()
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+
+    if form.validate_on_submit():
+        uri = form.uri.data
+        name = form.name.data
+        slugs = form.slugs.data
+        users = form.users.data
+
+        project = {
+            "uri": uri,
+            "name": name,
+            "slugs": re.split('\s?,\s?', slugs)
+        }
+
+        if users:
+            project['users'] = re.split('\s?,\s?', users)
```

Comment:
This only seems to add users to a project, without differentiating permissions levels. Is that all we want here?